### PR TITLE
DNX command line changes

### PR DESCRIPTION
**Pull request by TomArcher**

### CODE REVIEW

**Comment by TomArcher** [comment]
@Rick-Anderson @danroth27 @Tratcher @Erikre Please review

**Comment by danroth27** [comment]
Can we update the console app image to use beta7 instead of beta6?

**Comment by danroth27** [comment]
Also, can you add some text that clarifies

File: aspnet/dnx/console.rst [hunk 1]
```diff
@@ -45,11 +45,11 @@ Save your changes.
 Running the App
 ---------------
 
-At this point, we're ready to run the app. You can do this by simply entering ``dnx . run`` from the command prompt. You should see a result like this one:
+At this point, we're ready to run the app. You can do this by simply entering ``dnx run`` from the command prompt. You should see a result like this one:
 
 .. image:: console/_static/dnx-run.png
 
-.. note:: The ``dnx`` command is used to execute a managed entry point (a ``Program.Main`` function) in an assembly. The ``dnx . run`` command is a shorthand for executing the entry point in the current project. It is equivalent to ``dnx . [project_name]``
+.. note:: The ``dnx`` command is used to execute a managed entry point (a ``Program.Main`` function) in an assembly.
```
Comment (danroth27) [inline aspnet/dnx/console.rst]: Clarify that by default dnx looks in the current directory for the project to run. To specify a different directory use the --project switch.

**Comment by danroth27** [comment]
:watch:

**Comment by TomArcher** [comment]
@danroth27 You'll need to reshoot the image that requires the change from beta6 to beta7. I've made the other changes.

**Comment by danroth27** [comment]
:shipit:

**Pull request merged**